server/src/main/java/org/apache/iotdb/db/query/control/SessionTimeoutManager.java

```diff
@@ -30,14 +30,18 @@
 import java.util.concurrent.TimeUnit;
 
 public class SessionTimeoutManager {
-  private static final long CLEANUP_PERIOD_MILLIS = 10 * 1000; // TODO: Use a more reasonable value
+  private static final long SESSION_TIMEOUT =
+      IoTDBDescriptor.getInstance().getConfig().getSessionTimeoutThreshold();
+  private static final long CLEANUP_PERIOD_MILLIS = Math.max(5000, SESSION_TIMEOUT / 5);
   private static final Logger LOGGER = LoggerFactory.getLogger(SessionTimeoutManager.class);
 
-  private Map<Long, SessionInfo> idToSessionInfo;
+  private Map<Long, Long> idToLastActiveTime;
```

Comment:
Rename `sessionIdToLastActiveTime`?

Reply:
Lol I was about to change to the exact same name

Reply:
hhh It's cool :D